templates/conf.d/nginx.conf.erb

```diff
@@ -1,4 +1,11 @@
 # MANAGED BY PUPPET
+<% Array(@dynamic_modules).each do |mod_item| -%>
```

Comment:
Can you get rid of the Array()?
That isn't needed since it's always an array.

Reply:
I will do. Sorry, I am not very good with ERB